DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -338,80 +470,80 @@
 				<field name="" dataType="f32"/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="604" name="TexturesBlend(Texture, Texture, Texture, Animation)">
+		<ActionID ID="604" name="Textures Blend (Texture, Texture, Animated Texture)">
```

Comment:
This one is good, but the space in between blend and the parenthesis is slightly confusing. Makes it look like the 2 things are not connected.